app/pages/UcsfClientEligibilityPage/ucsfEligibilities.ts

```diff
@@ -0,0 +1,101 @@
+// We are hardcoding the eligibility data as it currently exists on the design specs.
+// When content has entered UCSF specific eligibilities, will we fetch that data from the API
+
+export interface ucsfEligibilityDictionary {
```

Comment:
```suggestion
export interface UcsfEligibilityDictionary {
```

Nit: Please use a capital initial letter for interfaces, since they are kind of like defining a class at the type level.